Tests/Units/EntityRepositoryTest.php

```diff
@@ -20,12 +20,12 @@
 use Symfony\Component\Cache\Adapter\ArrayAdapter;
 
 /**
- * @covers EntityRepository
+ * @covers \EntityRepository
```

Comment:
EntityRepository is not a first-level namespace

```suggestion
 * @covers EntityRepository
```

Reply:
It's just a fail with my IDE